rllib/evaluation/sampler.py

```diff
@@ -189,20 +184,24 @@ def __init__(
             render (bool): Whether to try to render the environment after each
                 step.
         """
+        if policies is not None:
+            deprecation_warning(old="policies")
+        if policy_mapping_fn is not None:
+            deprecation_warning(old="policy_mapping_fn")
```

Comment:
Will need documentation changes since Sampler is core part of RLlib

Reply:
Added comments that explain that we can use the `worker` arg, which is passed in as well, for all these 4 deprecated args. E.g. `policies = worker.policy_map` or `policy_mapping_fn = worker.policy_mapping_fn`.